src/main/java/com/jalasoft/webservice/Constants.java

```diff
@@ -0,0 +1,16 @@
+package com.jalasoft.webservice;
+
```

Comment:
Maybe, this class could be moved to a folder where different utilities or resources classes will be included on it, what do you think?

Reply:
done